docs/operating-scylla/nodetool-commands/cluster/repair.rst

```diff
@@ -0,0 +1,82 @@
+Nodetool cluster repair
+=======================
+
+**cluster repair** - A process that runs in the background and synchronizes the data between nodes. It only repairs keyspaces with tablets enabled (default).
+To repair keyspaces with tablets disabled (vnodes-based), see :doc:`nodetool repair </operating-scylla/nodetool-commands/repair/>`.
+
+Running ``cluster repair`` on a **single node** synchronizes all data on all nodes in the cluster.
+
+To synchronize all data in clusters that have both tablets- and vnodes-based keyspaces, run :doc:`nodetool repair -pr </operating-scylla/nodetool-commands/repair/>` on **all**
+of the nodes in the cluster, and :doc:`nodetool cluster repair </operating-scylla/nodetool-commands/cluster/repair/>` on  **any** of the nodes in the cluster.
+
+To check if a keyspace enables tablets, use:
+
+.. code-block:: cql
+
+  DESCRIBE KEYSPACE `keyspace_name`
+
+.. note:: Run the :doc:`nodetool cluster repair </operating-scylla/nodetool-commands/cluster/repair/>` command regularly. If you delete data frequently, it should be more often than the value of ``gc_grace_seconds`` (by default: 10 days), for example, every week.
```

Comment:
We'd recommend users to always use tombstone gc repair mode. When this mode is used, there is no need to run repair within gc_grace_seconds. The frequency of repair should not depend on gc_grace_seconds at all.